### PR TITLE
Update bigtable docs to reflect latest client

### PR DESCRIPTION
**Pull request by igorbernstein2**
This is to address the issues uncovered in this issue:
https://github.com/JanusGraph/janusgraph/issues/450#issuecomment-321024231

Please note, that there will be one more change before bigtable-hbase 1.0 is released that will remove the need for the netty-tcnative-boringssl-static dependency

### CODE REVIEW

**Comment by mbrukman** [comment]
Hey there @igorbernstein2, thanks for the PR! Since you're from Google, I can add you to our CCLA; please give me a few minutes.

**Comment by mbrukman** [comment]
While I'm working on the CCLA, @igorbernstein2, please follow [these instructions](https://github.com/JanusGraph/janusgraph/blob/master/CONTRIBUTING.md#commit-changes-and-sign-the-developer-certificate-of-origin) to add a `Signed-off-by` line to your commit:

> ```
> $ git commit --amend -s
> $ git push -f
> ```

**Comment by mbrukman** [comment]
Please also update the `docs/versions.txt` file to reflect this change in Bigtable client version.

File: docs/versions.txt [hunk 1]
```diff
@@ -13,9 +13,9 @@ JanusGraph.
 [options="header"]
 |==========================
 | JanusGraph | Cassandra | HBase | Bigtable | Elasticsearch | Solr | TinkerPop | Spark | Scala
-| 0.1.0 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.z | 1.5.z |  5.2.z | 3.2.z | 1.6.z | 2.10.z
-| 0.1.1 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.z | 1.5.z |  5.2.z | 3.2.z | 1.6.z | 2.10.z
-| 0.2.0 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.z | 1.z*,2.z,5.z |  5.3-5.5.z, 6.2-6.6.z | 3.2.z | 1.6.z | 2.10.z
+| 0.1.0 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.7.1, 1.0.0-preZ | 1.5.z |  5.2.z | 3.2.z | 1.6.z | 2.10.z
+| 0.1.1 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.7.1, 1.0.0-preZ | 1.5.z |  5.2.z | 3.2.z | 1.6.z | 2.10.z
```
Comment (sjudeng) [inline docs/versions.txt]: Is previously released 0.1.0 and 0.1.1 really compatible with 1.0.0-preZ Bigtable?

Comment (igorbernstein2) [inline docs/versions.txt]: bigtable-hbase simply implements the hbase 1.x api. The internal implementation should not affect consumers of the hbase api.

File: docs/versions.txt [hunk 2]
```diff
@@ -13,9 +13,9 @@ JanusGraph.
 [options="header"]
 |==========================
 | JanusGraph | Cassandra | HBase | Bigtable | Elasticsearch | Solr | TinkerPop | Spark | Scala
-| 0.1.0 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.z | 1.5.z |  5.2.z | 3.2.z | 1.6.z | 2.10.z
-| 0.1.1 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.z | 1.5.z |  5.2.z | 3.2.z | 1.6.z | 2.10.z
-| 0.2.0 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.z | 1.z*,2.z,5.z |  5.3-5.5.z, 6.2-6.6.z | 3.2.z | 1.6.z | 2.10.z
+| 0.1.0 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.7.1, 1.0.0-preZ | 1.5.z |  5.2.z | 3.2.z | 1.6.z | 2.10.z
```
Comment (mbrukman) [inline docs/versions.txt]: We shouldn't change this retroactively to claim to not support the versions we supported earlier. Namely, JanusGraph previously shipped claiming to support Bigtable 0.9.5.z (and specified 0.9.z as the generic format); with the change to support literally "0.9.7.1", it implies that 0.9.5.z is/was no longer supported.

My recommendation is to change it back to 0.9.z, while adding 1.0.0-preZ as an additionally-supported version mask.

Thoughts?

**Comment by igorbernstein2** [comment]
updated

**Comment by mbrukman** [comment]
@sjudeng – can you please take another look at the PR?

**Comment by mbrukman** [comment]
Also, @igorbernstein2, if you plan to either squash this PR into a single commit and re-push, or for future PRs: for documentation-only changes such as this one, please include `[skip ci]` on the first line of your commit so that we don't spend Travis CI cycles on changes which touch no code, as it will conserve them for our other PRs. Thanks!

**Comment by igorbernstein2** [comment]
squashed & rebased

**Comment by mbrukman** [comment]
Thanks again, @igorbernstein2!

**Pull request merged**